webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -243,17 +253,33 @@ const SubscribeModal = () => {
                 {
                     !showResultPanel && (
                         isAnyProjectLinked ? (
-                            Object.keys(subscriptionModalFields).map((field) => (
-                                <Form
-                                    key={subscriptionModalFields[field as SubscriptionModalFields].label}
-                                    fieldConfig={subscriptionModalFields[field as SubscriptionModalFields]}
-                                    value={formFields[field as SubscriptionModalFields] ?? ''}
-                                    optionsList={getDropDownOptions(field as SubscriptionModalFields)}
-                                    onChange={(newValue) => onChangeFormField(field as SubscriptionModalFields, newValue)}
-                                    error={errorState[field as SubscriptionModalFields]}
-                                    isDisabled={isLoading}
-                                />
-                            ))
+                            <>
+                                {
+                                    Object.keys(subscriptionModalFields).map((field) => (
+                                        <Form
+                                            key={subscriptionModalFields[field as SubscriptionModalFields].label}
+                                            fieldConfig={subscriptionModalFields[field as SubscriptionModalFields]}
+                                            value={formFields[field as SubscriptionModalFields] ?? ''}
```

Comment:
can formFields[...] be undefined or null? If not, then I think we can skip `?? ''`

Reply:
yes, they can be undefined especially when the API call is in progress or there doesn't exist a default value for a field.